docs/reference/announcements.md

```diff
@@ -132,6 +132,31 @@ Starting with Camunda 8.6, the Camunda Optimize artifact has been split into two
 
 Make sure to update your Docker configurations accordingly to ensure compatibility.
 
+### New base path for Operate and Tasklist web applications
+
+We are introducing a new base path for both the Operate and Tasklist **web applications**. This change applies to both Self-Managed and SaaS environments.
```

Comment:
My understanding is that we will move back to the original endpoints in 8.7. We should consider phrasing in a way to indicate this.

Reply:
@theburi I think your are referring to https://github.com/camunda/camunda/issues/23021 ?
I think it will not make it for 8.7.